go/test/endtoend/vreplication/resharding_workflows_v2_test.go

```diff
@@ -39,7 +39,7 @@ const (
 	reverseKsWorkflow = sourceKs + "." + workflowName + "_reverse"
 	tablesToMove      = "customer"
 	defaultCellName   = "zone1"
-	readQuery         = "select * from customer"
+	readQuery         = "select cid from customer"
```

Comment:
Why is this change required?

Reply:
the test expectation was not matching as the actual query getting executed is the expanded one with cid instead of `star`

This is what I can understand from Rohit's commit message.